src/x/headers/headers.go

```diff
@@ -43,6 +43,10 @@ const (
 	// Valid values are "unaggregated" or "aggregated".
 	MetricsTypeHeader = M3HeaderPrefix + "Metrics-Type"
 
+	// PromTypeHeader sets the prometheus metric type. Valid values are
+	// "counter", "gauge", etc.
```

Comment:
Nit: add a link to the code that has those values listed so that there is no need for "etc".